.github/workflows/ci.yml

```diff
@@ -26,6 +26,17 @@ jobs:
           lsblk -l
           echo "sda rotational = $(cat /sys/block/sda/queue/rotational)"
           echo "sdb rotational = $(cat /sys/block/sdb/queue/rotational)"
+      - name: Create virtual environment
```

Comment:
I meant that you need to change the current Install dependencies step line 22 to run in virtual env

Reply:
yep yep, I was distracted with the gh runner :stuck_out_tongue: